### DISCUSSION TITLE
OpenOil Canary Islands

### DISCUSSION THREAD

**Discussion by Alvarnb3** [OpenDrift/opendrift#1253] [Q&A] [answered]
Hello! I am a new user of OpenDrift and I can not find opendrift.readers for the canary islands territory. Can someone help please!!

**Comment by knutfrode** [comment, marked as answer]
Hi,

There are some datasets listed here:
https://github.com/OpenDrift/opendrift/blob/master/opendrift/scripts/data_sources.txt

of which a few are global:

Wind:
https://pae-paha.pacioos.hawaii.edu/thredds/dodsC/ncep_global/NCEP_Global_Atmospheric_Model_best.ncd

Currents:
https://nrt.cmems-du.eu/thredds/dodsC/cmems_mod_glo_phy_anfc_merged-uv_PT1H-i
https://tds.hycom.org/thredds/dodsC/GLBy0.08/latest
https://tds.hycom.org/thredds/dodsC/GLBy0.08/expt_93.0/uv3z

You can check coverage in time and space and available variables by creating and printing a reader, or commandline `readerinfo <URL/file>`
Note that all these forcing datasets are independent of OpenDrift, and you may find more by searching the internet or contacting relevant institutes.
